src/Gui/View3DInventorViewer.cpp

```diff
@@ -3341,6 +3342,49 @@ void View3DInventorViewer::viewSelection()
     }
 }
 
+void View3DInventorViewer::alignToSelection()
+{
+    if (!getCamera()) {
+        return;
+    }
+
+    const auto selection = Selection().getSelection();
+
+    // Empty selection
+    if (selection.empty()) {
+        return;
+    }
+
+    // Too much selections
+    if (selection.size() > 1) {
+        return;
+    }
+
+    const App::SubObjectT selected(selection[0].pObject, selection[0].SubName);
+
+    Base::PyGILStateLocker lock;
+
+    PyObject* pyObj = nullptr;
+    Base::Matrix4D matrix;
+    selected.getObject()->getSubObject(selected.getSubName().c_str(), &pyObj, &matrix);
```

Comment:
Here a new Python object is created but its counter is not decremented which causes a memory leak.

Reply:
Python object is no longer used